packages/ui/LabeledIcon/LabeledIcon.js

```diff
@@ -39,7 +39,7 @@ const LabeledIconBase = kind({
 		 * @type {Node}
 		 * @public
 		 */
-		children: PropTypes.node,
+		children: PropTypes.node.isRequired,
```

Comment:
Would it make more sense to make the `icon` property required rather than the `children` property?